test/jdk/java/net/DatagramSocket/SendCheck.java

```diff
@@ -126,11 +125,8 @@ Object[][] providerIO() throws IOException {
 
         List<Sender> senders = List.of(
                 Sender.of(new DatagramSocket(null)),
-                Sender.of(new MulticastSocket(null), (byte) 0),
                 Sender.of(DatagramChannel.open()),
-                Sender.of(DatagramChannel.open().socket()),
-                Sender.of((MulticastSocket)
-                        DatagramChannel.open().socket(), (byte) 0)
```

Comment:
Should we instead just remove the ttl parameter and test with plain MulticastSocket::send?

Reply:
That's a good catch. The test description of this test states that it's meant to test DatagramSocket, the adaptor, the DatagramChannel and the MulticastSocket. The change I did would have stopped testing MulticastSocket altogether. I have now updated the PR to only remove the use of a setTTL() from this test and it now continues to test the send() methods of these classes.